.github/workflows/integration-tests.yaml

```diff
@@ -95,11 +95,15 @@ jobs:
 
       - name: Run tests
         run: |
-          pip install pytest
+          pip install pytest urllib3 huggingface_hub
           python3 -m pip install -e sdk/python; pytest -s sdk/python/test --log-cli-level=debug --namespace=default
         env:
           GANG_SCHEDULER_NAME: ${{ matrix.gang-scheduler-name }}
 
+      - name: Run specific tests for Python 3.10+
```

Comment:
Since `match` is released in [python 3.10](https://docs.python.org/3/whatsnew/3.10.html#summary-release-highlights), I created another step for the e2e.

Reply:
Where do you use match in the tests ?

Reply:
I didn't use `match` in the tests. `match` is used in https://github.com/kubeflow/training-operator/blob/master/pkg/initializer_v2/model/__main__.py#L23.

Reply:
Oh, good point.
Let's actually use the same Python version that we use in our initializer images: https://github.com/kubeflow/training-operator/blob/master/cmd/initializer_v2/dataset/Dockerfile#L1.
E.g. Python 3.11